### PR TITLE
support kafka 2.1.1

### PR DESCRIPTION
**Pull request by stanlyDoge**
### Checklist

- [ ] Update/write design documentation in `./design`
- [ ] Write tests
- [ ] Make sure all tests pass
- [ ] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

File: documentation/common/version-dependent-attrs.adoc [hunk 1]
```diff
@@ -1,3 +1,3 @@
 // Generated by documentation/version-dependent-attrs.sh during the build
 // DO NOT EDIT BY HAND
-:DefaultKafkaVersion: 2.1.0
+:DefaultKafkaVersion: 2.1.1
```
Comment (laidan6000) [inline documentation/common/version-dependent-attrs.adoc]: Are you sure it's OK to edit this file manually?

Comment (stanlyDoge) [reply documentation/common/version-dependent-attrs.adoc]: I run `make all` command, which edited this value.

File: olm/strimzi-cluster-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -5,8 +5,7 @@ metadata:
...
...{"secretName":"my-cluster-cluster-ca-cert","certificate":"ca.crt"}]}}}, {"apiVersion":"kafka.strimzi.io/v1alpha1","kind":"KafkaTopic","metadata":{"name":"my-topic","labels":{"strimzi.io/cluster":"my-cluster"}},"spec":{"partitions":10,"replicas":3,"config":{"retention.ms":604800000,"segment.bytes":1073741824}}}, {"apiVersion":"kafka.strimzi.io/v1alpha1","kind":"KafkaUser","metadata":{"name":"my-user","labels":{"strimzi.io/cluster":"my-cluster"}},"spec":{"authentication":{"type":"tls"},"authorization":{"type":"simple","acls":[{"resource":{"type":"topic","name":"my-topic","patternType":"literal"},"operation":"Read","host":"*"},{"resource":{"type":"topic","name":"my-topic","patternType":"literal"},"operation":"Describe","host":"*"},{"resource":{"type":"group","name":"my-group","patternType":"literal"},"operation":"Read","host":"*"},{"resource":{"type":"topic","name":"my-topic","patternType":"literal"},"operation":"Write","host":"*"},{"resource":{"type":"topic","name":"my-topic","patternType":"literal"},"operation":"Create","host":"*"},{"resource":{"type":"topic","name":"my-topic","patternType":"literal"},"operation":"Describe","host":"*"}]}}}]    categories: "kafka,messaging,streaming"
```
Comment (stanlyDoge) [inline olm/strimzi-cluster-operator.clusterserviceversion.yaml]: This seems crazy but it is just abou tadding `"version"` to kafka resources (Kafka, Connect, ConnectS2I)

Comment (scholzj) [reply olm/strimzi-cluster-operator.clusterserviceversion.yaml]: I think you have it wrong ... you seem to use `"version:2.1.1"` ... but it should probably be `"version": 2.1.1` or `"version": "2.1.1"` ;-)

Comment (stanlyDoge) [reply olm/strimzi-cluster-operator.clusterserviceversion.yaml]: I used the same format as for 
```spec:
  kafka:
    version: 2.1.1
    replicas: 3
...``` 
---> `"spec":{"kafka":{"version:2.1.1","replicas":3,...`

Comment (scholzj) [reply olm/strimzi-cluster-operator.clusterserviceversion.yaml]: You have the `"` wrong.

Comment (stanlyDoge) [reply olm/strimzi-cluster-operator.clusterserviceversion.yaml]: Ahh, good catch!

**Pull request merged**